apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -402,6 +422,37 @@ export class DotEditContentHtmlService {
         this.dotEditContentToolbarHtmlService.addContainerToolbar(doc);
     }
 
+    private injectInlineEditingScripts(): void {
+        const doc = this.getEditPageDocument();
+        const TINYMCE = `/html/js/tinymce/js/tinymce/tinymce.min.js`;
+        const tinyMceScript: HTMLScriptElement = this.dotDOMHtmlUtilService.creatExternalScriptElement(
+            TINYMCE
+        );
+
+        this.initInlineEditing();
+        doc.body.append(tinyMceScript);
+    }
+
+    private initInlineEditing(): void {
+        const doc = this.getEditPageDocument();
+        const tinyMceInit: HTMLScriptElement = this.dotDOMHtmlUtilService.createInlineScriptElement(
+            INLINE_TINYMCE_SCRIPTS
+        );
+
+        this.dotLicenseService
+            .isEnterprise()
+            .pipe(take(1))
+            .subscribe((isEnterprise) => {
+                if (isEnterprise) {
+                    doc.body.append(tinyMceInit);
+                    const editModeNode = doc.querySelectorAll('[data-mode]');
+                    editModeNode.forEach((node) => {
+                        node.classList.add('edit-mode');
```

Comment:
`edit-mode` is a very common class name that can be override by external css, let's use something like: `dotcms__inline-edit-field`